api/nomics.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import calendar
+import connexion
+
+from services.bitshares_websocket_client import client as bitshares_ws_client
+from services.bitshares_elasticsearch_client import client as bitshares_es_client
+from services.cache import cache
+import api.explorer
+import es_wrapper
+import config
+
+
+def info():
+    return {
+        "name": "Bitshares",
+        "description": "An exchange description of at least 1000 characters in plain text (no html)",
+        "location": "Anywhere",
+        "logo": "https://example.com/exchange-logo.png",
+        "website": "https://bitshares.org/",
+        "twitter": "example",
+        "capability": {
+            "markets": True,
+            "trades": True,
+            "tradesSocket": False,
+            "orders": False,
+            "ordersSocket": False,
+            "ordersSnapshot": True,
+            "candles": False
+        }
+    }
+
+
+def markets():
+    result = []
+    top_markets = bitshares_ws_client.request('database', 'get_top_markets', [100])
+
+    for market in top_markets:
+        result.append({
+            'id': market["base"] + "_" + market["quote"],
+            'base': market["base"],
+            'quote': market["quote"]
+        })
+    return result
+
+
+@cache.memoize()
+def trades(market, since):
```

Comment:
The implementation would be definitively better with ES.
`since` match the `search_after` ES feature quite well.

Reply:
Changed `trades` call to Elastic in commit: https://github.com/bitshares/bitshares-explorer-api/pull/64/commits/9ba6788024a65a6149cf0adc63fd766baa0e3c24

Demo is up at: http://185.208.208.184:5000/apidocs/#!/nomics/api_nomics_trades